orga/tests/unit/controllers/authenticated/certifications_test.js

```diff
@@ -143,4 +143,89 @@ module('Unit | Controller | authenticated/certifications', function(hooks) {
       assert.notOk(controller.isCertificationAttestationDownloadEnabled);
     });
   });
+
+  module('#downloadAttestation', function() {
+
+    test('should call the pdf file-saver service with the right parameters', async function(assert) {
+      // given
+      const controller = this.owner.lookup('controller:authenticated/certifications');
+
+      const token = 'a token';
+      const fileName = 'attestations_pix.pdf';
+      const organizationId = 12345;
+      const selectedDivision = '3èmea';
+
+      controller.selectedDivision = selectedDivision;
+
+      controller.session = {
+        isAuthenticated: true,
+        data: {
+          authenticated: {
+            access_token: token,
+          },
+        },
+      };
+
+      controller.currentUser = {
+        organization: {
+          id: organizationId,
+        },
+      };
+
+      controller.fileSaver = {
+        save: sinon.stub(),
+      };
+
+      controller.model = {
+        options: [{ label: '3èmeA', value: '3èmeA' }],
+      };
+
+      const event = {
+        preventDefault: sinon.stub(),
+      };
+
+      // when
+      await controller.downloadAttestation(event);
+
+      // then
+      assert.ok(controller.fileSaver.save.calledWith(
+        {
+          token,
+          fileName,
+          url: `/api/organizations/${organizationId}/certification-attestations?division=${selectedDivision}`,
+        },
+      ));
+    });
+
+    test('it should not call downloadAttestation service and display an error if division is invalid', async function(assert) {
```

Comment:
it should not call ~~downloadAttestation~~ fileSaver service and display an error if division is invalid